workloads.tex

```diff
@@ -68,21 +65,25 @@ \subsection{Graph Patterns}
 
 To illustrate queries, we use graph patterns such as \autoref{fig:example-graph-pattern} with the following notation:
 
-\begin{itemize}
-	\item Filtering conditions are typeset in \textit{italic}.
-	\item Properties that should be returned are denoted in normal (book) font.
-	\item Negative conditions, i.e., edges that are now allowed in the graph are denoted with \textcolor{red}{\dashuline{dashed red}} lines.
-	\item Aggregations are shown in dashed boxes.
-\end{itemize}
-
 \begin{figure}[ht]
 	\begin{center}
-		\includegraphics[scale=\patternscale,margin=0cm .2cm]{patterns/bi-read-23}
+		\includegraphics[scale=\patternscale,margin=0cm .2cm]{patterns/bi-read-08}
 		\caption{Example graph pattern.}
 		\label{fig:example-graph-pattern}
 	\end{center}
 \end{figure}
 
+\begin{itemize}
+	\item Vertices are . If the value of the vertex is part of the query result, it is denoted on the figure as 
```

Comment:
This line is wrong. Fix it

Reply:
Fixed in https://github.com/ldbc/ldbc_snb_docs/pull/81/commits/65e0a405917e507280bbd44f385550188b2a4935